### PR TITLE
feat: Added "Reset to Defaults" button to EFB throttle calibration page

### LINKED ISSUE #6113: Option to reset Throttle Calibration Values
### Aircraft Version

Development

### Description

To make it easier to for users to start with a fresh set of default values and also to make it easier for #support we should have a button to reset the calibration values to the default values. 

This would spare the user and support to have to remove ThrottleCalibration.ini (and restart the sim) when the user managed to somehow mess up the calibration. 

DM me for questions. 

### References (optional)

n/a

### Additional info (optional)

n/a

### Discord Username (optional)

Cdr_Maverick#6475

### PR DESCRIPTION
**Pull request by frankkopp**
Fixes #6113 

## Summary of Changes
This PR adds a button to reset the current throttle calibration values to the default values.

## Screenshots (if necessary)
image

## Additional context
This PR adds a new event A32NX_THROTTLE_MAPPING_SET_DEFAULTS and and new function in ThrottleAxisMapping.h/cpp to reset the current values for the calibration to their default values as defined in 

```
ThrottleAxisMapping::Configuration ThrottleAxisMapping::getDefaultConfiguration() {
  return {
      true,  // use reverse on axis
      -1.00,  // reverse low
      -0.95,  // reverse high
      -0.72,  // reverse idle low
      -0.62,  // reverse idle high
      -0.50,  // idle low
      -0.40,  // idle high
      -0.03,  // climb low
      +0.07,  // climb high
      +0.42,  // flex/mct low
      +0.52,  // flex/mct high
      +0.95,  // toga low
      +1.00   // toga high
  };
}
```

After resetting the user can start over with calibration and safe the calibration with the Safe button as before. 

Discord username (if different from GitHub): Cdr_Maverick#6475

## Testing instructions
1. Open EFB Throttle Calibration page - confirm new "Reset to Defaults" button is there and active
2. Change your config - mess it up - and use the button to reset the config to default values (see values above)
3. Apply & Save and check that the configuration is saved - also by restart the flight and re-starting the sim

Be aware the "Independent Axis" is not part of the saved configuration. This will not reset with the defaults. 

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page

### CODE REVIEW

File: docs/a320-events.md [hunk 1]
```diff
@@ -143,7 +143,10 @@
 
 ## Throttle Mapping System
 
-- A32NX.THROTTLE_MAPPING_LOAD_FROM_FILE
+- A32NX.THROTTLE_MAPPING_SET_DEFAULTS
+    - When triggered the throttle mapping configuration resets to default values
+
+  - A32NX.THROTTLE_MAPPING_LOAD_FROM_FILE
     - When triggered the throttle mapping configuration is loaded from **file**
```
Comment (Saschl) [inline docs/a320-events.md:150]:
```suggestion
 - A32NX.THROTTLE_MAPPING_LOAD_FROM_FILE
     - When triggered the throttle mapping configuration is loaded from **file**
```

File: src/fbw/src/ThrottleAxisMapping.cpp [hunk 2]
```diff
@@ -77,6 +77,22 @@ bool ThrottleAxisMapping::loadFromLocalVariables() {
   return true;
 }
 
+bool ThrottleAxisMapping::setDefaults() {
+  // read configuration from file or use default
```
Comment (Saschl) [inline src/fbw/src/ThrottleAxisMapping.cpp:81]: No file here,  I guess we can remove this comment at this place ;)

File: src/fbw/src/ThrottleAxisMapping.cpp [hunk 3]
```diff
@@ -77,6 +77,22 @@ bool ThrottleAxisMapping::loadFromLocalVariables() {
   return true;
 }
 
+bool ThrottleAxisMapping::setDefaults() {
```
Comment (Saschl) [inline src/fbw/src/ThrottleAxisMapping.cpp:80]: suggestion: `applyDefaults()` might describe the action a bit better as we don't set/change defaults but apply the to the config. Very subjective though so proceed as you wish :)

**Review by Saschl — APPROVED** [review]
Just some minor things, not affecting functionality. Good work!

**Review by Benjozork — APPROVED** [review]

**Comment by ShadowFrost3** [comment]
Quality Assurance Tester Report


Object of testing: #6393
Tier of Testing:1
Date: 29/12/2021

Testing Process:
Reset profile via button
Created new profile 
saved and flew aircraft as expected
imported old profile 
verified that imported profile stayed through MSFS restart 

Negatives: 
Testing Results: Passed

**Comment by FlightSimmer7700** [comment]
Quality Assurance Tester

Discord: FlightSimmer7700#5438
Object of testing: #6393 
Tier of Testing: 1
Date: 31/12/21

Testing Process: 
Loaded in C&D and tested my profile
Reset, saved, restarted flight and tested
Reset, didn't save and restarted flight
Tested in mid air
Negatives: N/A
Testing Results: Passed

Conclusions: LGTM! (have not tested after latest change of independent axis)

File: src/fbw/src/interface/SimConnectInterface.cpp [hunk 4]
```diff
@@ -1892,6 +1890,12 @@ void SimConnectInterface::simConnectProcessEvent(const SIMCONNECT_RECV_EVENT* ev
       break;
     }
 
+    case Events::A32NX_THROTTLE_MAPPING_SET_DEFAULTS: {
+      cout << "WASM: event triggered: THROTTLE_MAPPING_SET_DEFAULTS" << endl;
+      throttleAxis[0]->applyDefaults();
+      throttleAxis[1]->applyDefaults();
+      break;
+    }
     case Events::A32NX_THROTTLE_MAPPING_LOAD_FROM_FILE: {
```
Comment (aguther) [inline src/fbw/src/interface/SimConnectInterface.cpp:1899]:
```suggestion

    case Events::A32NX_THROTTLE_MAPPING_LOAD_FROM_FILE: {
```

**Review by aguther — APPROVED** [review]

**Pull request merged**